topic-operator/src/test/java/io/strimzi/operator/topic/TopicOperatorBaseIT.java

```diff
@@ -82,7 +87,11 @@ public void run() {
             }
         }
     };
-    protected final long timeout = 30_000L;
+    protected final long timeout = 600_000L;
```

Comment:
That is fairly long timeout ... is that intentional or just a remain of some tests?

Reply:
No, that's intentional. It's actually to cover deletion. What I observed (repeatedly, but not 100% reproducibly) was that deletion of the `KafkaTopic` can sometimes take >5 minutes. The `KafkaTopic` gets a `deletionTimestamp` immediately, but then it can be a long while before we observe the delete event. Because we delete topics `@After` the tests this was causing failures. We have to delete topics after tests though, since without it we don't have the necessary isolation between tests and they fail in other ways.

Reply:
Ok, good. It caught my attention but if it is intentional then OK.